### ISSUE TITLE
Acts 15:16-17 to reflect quoted OT scripture Amos 9:11-12

### ISSUE THREAD

**Issue by Ekkles** [openenglishbible/Open-English-Bible#303]
[Acts 15:16-17] substitutes 'Gentiles' for nations in [Amos 9:11-12].

American Standard Version has [Amos 9:12] as ".. that they may possess the remnant of Edom, and all the nations that are called by my name, saith Jehovah that doeth this" which is the verse being quoted in [Acts 15:16-17] where  גוי (gowy H1471) is translated as 'nations'.

Translating ἔθνος (ethnos G1484) as nations, rather than Gentiles, in [Acts 15:17] reflects not only the primary meaning of the word ἔθνος but also correctly echo's the scripture being quoted (as is done by Aramaic Bible in Plain English, Douay-Rheims, Darby and Weymouth).

Such a translation choice suggests that the Greek translator were unaware of the earlier Hebrew translation.  Generally where the OT is quoted in the NT the translation choices should exhibit awareness of the text being quoted and make similar translation choices.

**Comment by zaddok** [comment]
It is not clear to me what the difference is between translating the Greek/Hebrew as "nations" vs "gentiles" would be. Are the English definitions of the terms "Gentile" and "Nation" different enough that this is a point of debate?

I wonder if you can clarify what practical implication this change has?
